swarm/storage/filestore.go

```diff
@@ -83,16 +85,29 @@ func NewFileStore(store ChunkStore, params *FileStoreParams) *FileStore {
 // It returns a reader with the chunk data and whether the content was encrypted
 func (f *FileStore) Retrieve(ctx context.Context, addr Address) (reader *LazyChunkReader, isEncrypted bool) {
 	isEncrypted = len(addr) > f.hashFunc().Size()
-	getter := NewHasherStore(f.ChunkStore, f.hashFunc, isEncrypted)
+	tag, err := f.tags.GetContext(ctx)
```

Comment:
could we rename `tags.GetContext` to `tags.NewFromContext`?

Reply:
it should actually be `GetFromContext`. renaming accordingly. it doesn't create a new one if it doesn't find one in the context, hence `NewFromContext` is incorrect